src/frontend/Komponenter/Behandling/VedtakOgBeregning/Felles/konstanter.ts

```diff
@@ -1 +1,9 @@
 export const VEDTAK_OG_BEREGNING = 'vedtak-og-beregning';
+
+// TODO: Legge til satser for flere år tilbake i tid
+// Det første elementet i listen tilsvarer satsen for 1 barn, det andre elementet for 2 barn og det tredje elementet for 3 barn
```

Comment:
Kunne vi hentet denne på samme måte som vi henter beløpFørSatsjustering? Litt kjiopt å holde dette oppdatert to steder?

Reply:
Se kommentar under.